llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -66,6 +66,8 @@ def riscv_sret_glue : SDNode<"RISCVISD::SRET_GLUE", SDTNone,
                              [SDNPHasChain, SDNPOptInGlue]>;
 def riscv_mret_glue : SDNode<"RISCVISD::MRET_GLUE", SDTNone,
                              [SDNPHasChain, SDNPOptInGlue]>;
+def riscv_mnret_glue : SDNode<"RISCVISD::MNRET_GLUE", SDTNone,
```

Comment:
This should be in another patch for CodeGen. And, do we really need it? It will be used in supervisor IIUC.

Reply:
RISCVISD::MNRET_GLUE isn't defined in this patch so it should definitely be removed.

Reply:
Thanks, I have removed the relevant content.